cmd/necogcp/cmd/createrunner.go

```diff
@@ -0,0 +1,195 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"text/template"
+
+	"github.com/cybozu-go/log"
+	"github.com/cybozu-go/neco-gcp/pkg/gcp"
+	"github.com/cybozu-go/neco-gcp/pkg/github"
+	"github.com/cybozu-go/well"
+	"github.com/spf13/cobra"
+	"google.golang.org/api/googleapi"
+)
+
+const (
+	SERVICE_ACCOUNT = "bootstrap-dctest@neco-test.iam.gserviceaccount.com"
```

Comment:
The project `neco-test` is hard-coded in the name of service account.
It is better to use the project specified in the command option.